From ac19a1a9b9408455b0796aa2410d41d9f2514a1c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20=C4=8Cerm=C3=A1k?= <dcermak@suse.com>
Date: Thu, 3 Nov 2022 15:47:22 +0100
Subject: [PATCH 2/2] Use the root group for the synced folders

The wheel group does not exist by default in opensuse/Tumbleweed and requiring
it breaks the tests.
---
 test/misc/Vagrantfile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/misc/Vagrantfile b/test/misc/Vagrantfile
index 92c4d16..f4115cb 100644
--- a/test/misc/Vagrantfile
+++ b/test/misc/Vagrantfile
@@ -14,7 +14,7 @@ Vagrant.configure(2) do |config|
         type: "sshfs",
         mount_options: ['ro', 'default_permissions'],
         owner: "root",
-        group: "wheel"
+        group: "root"
 
     # Test a forward mount to a location that is a symbolic link
     # https://github.com/dustymabe/vagrant-sshfs/issues/44
@@ -37,7 +37,7 @@ Vagrant.configure(2) do |config|
         type: "sshfs",
         reverse: true,
         owner: "root",
-        group: "wheel",
+        group: "root",
         mount_options: ['ro']
 
     host = 'sshfs-tests'
-- 
2.38.1

